server/game/GameActions/DealDamageAction.js

```diff
@@ -4,6 +4,7 @@ class DealDamageAction extends CardGameAction {
     setDefaultProperties() {
         this.amount = null;
         this.amountForCard = (card, context) => 1; // eslint-disable-line no-unused-vars
+        this.reduceArmorBy = (card, context) => 0; // eslint-disable-line no-unused-vars
```

Comment:
the eslint disabling is not needed here

Reply:
When I compiled, I got the context not used error.

Reply:
Ah yes, just remove the variables from both, they're not needed

Reply:
ok, I will make the change

Reply:
I get the same error on card:
C:\workspace_svn\public\keyteki\br-bugs\server\game\GameActions\DealDamageAction.js
  6:31  error  'card' is defined but never used  no-unused-vars
  7:31  error  'card' is defined but never used  no-unused-vars